pkg/constants.go

```diff
@@ -0,0 +1,14 @@
+package types
```

Comment:
imo these aren't types, they are constants or params. How about `pkg/params/constants.go`

Reply:
ok i will do this

Reply:
ok i have done this